index.ts

```diff
@@ -640,6 +640,10 @@ export default class FreeSound {
     return this.makeRequest<AccessTokenResponse>(postUrl, 'POST', data);
   }
 
+  /**
+   * This method allows searching sounds in Freesound by matching their tags and other
```

Comment:
nit: the `This method` prefix seems a bit unnecessary. Ideally we could find a way to omit it from all the doc comments. As a suggestion, maybe "Search sounds in Freesound by ..." might work. Feel free to experiment tho.

Reply:
I see. That makes sense.